### ISSUE TITLE
Annealed Temperature in Gumbel Softmax

### ISSUE THREAD

**Issue by jramapuram** [ethanfetaya/NRI#14]
It doesn’t look like the temperature is annealed in your gumbel softmax. Is there a reason for this as it is not standard? @tkipf

**Comment by ethanfetaya** [comment]
Mostly for simplicity and because it worked well without. Adding annealing would add the extra annealing hyperparameters to tune. If you add and see any noticeable improvement please let us know!
